doc/manifest-tool-guide.md

```diff
@@ -28,10 +28,9 @@ Sample output:
   paamfhAn5m1NM4UUu5mmvVaHQy8Fb65bkpbaNrvKwX3YMKdjzi2
 ```
 
-The first value is the master public key. Add the public key to the config
-for this validator. A one-word comment must be added after the key (for example
-*ThisServersName*). Any other rippled trusting the validator needs to add the
-master public key to its config. Only add keys received from trusted sources.
+The first value is the master public key. Any other rippled trusting the
+validator needs to add the master public key to its config. Only add keys
+received from trusted sources.
 
 The second value is the corresponding master secret key. **DO NOT INSTALL THIS
```

Comment:
We should consider changing the output of the manifest tool to include this warning, and format it in a way that makes it less likely for someone to copy-paste the whole thing in the config file accidentally.

For example:

 ```
### MASTER SECRET KEY
### PLEASE SAFEGUARD THE MASTER SECRET KEY AND DO NOT
### INSTALL IT IN THE CONFIG FILE.
###
### [master_secret]
### paamfhAn5m1NM4UUu5mmvVaHQy8Fb65bkpbaNrvKwX3YMKdjzi2

### MASTER PUBLIC KEY
[validator_keys]
nHDwNP6jbJgVKj5zSEjMn8SJhTnPV54Fx5sSiNpwqLbb42nmh3Ln
```